docs/source/peers/peers.md

```diff
@@ -1,193 +1,205 @@
-
-Other elements of the blockchain network are of course important: ledgers and
-smart contracts, orderers, policies, channels, applications, organizations,
-identities, and membership, and you can read more about them in their own
-dedicated sections. This section focusses on peers, and their relationship to those
-other elements in a Fabric network.
+A fundamental element of a Hyperledger Fabric blockchain network is the set of
+*peer nodes* (or, simply, *peers*). Peers are fundamental because they manage
+ledgers and smart contracts. Starting in Hyperledger Fabric v2.4, peers also
+manage transaction proposals and endorsements by running the
+[Fabric Gateway](../gateway.html) service. Recall that a ledger
+immutably records all of the transactions generated by smart contracts (which in
+  Hyperledger Fabric are contained
+in *chaincode*, more on this later) and endorsed by the required organizations.
+Smart contracts and ledgers encapsulate the *processes* and *information*, respectively,
+that are shared by channel peers. These aspects of peers make them a good starting
+point for understanding a Fabric network.
+
+Besides peers, other elements of a Fabric blockchain network are also important:  
+ledgers and smart contracts, orderers, endorsement policies, channels, client applications,
+organizations, identities, and membership, which you can read about in their own dedicated sections.
+This section focuses on peers, and their relationships to these other elements in a Fabric
+network.
 
 ![Peer1](./peers.diagram.1.png)
 
-*A blockchain network is comprised of peer nodes, each of which can hold copies
-of ledgers and copies of smart contracts. In this example, the network N
-consists of peers P1, P2 and P3, each of which maintain their own instance of
-the distributed ledger L1. P1, P2 and P3 use the same chaincode, S1, to access
-their copy of that distributed ledger*.
+*A Fabric blockchain network (above) is comprised of peers (non-ordering nodes), each
+of which stores and manages copies of ledgers and smart contracts. In this example, the Fabric
+network N consists of peers P1, P2 and P3, each of which maintains its own instance of
+the distributed ledger L1. P1, P2 and P3 each invoke the same chaincode, S1,
+to access their respective copies of the distributed ledger.*
 
-Peers can be created, started, stopped, reconfigured, and even deleted. They
-expose a set of APIs that enable administrators and applications to interact
-with the services that they provide. We'll learn more about these services in
-this section.
+Peers are a flexible and redundant element that can be created, started,
+stopped, reconfigured and deleted. Peers expose a set of APIs that enable administrators
+and client applications to interact with the services that peers provide --- the
+[Fabric Gateway](../gateway.html) service in particular.
 
-### A word on terminology
+## Fabric Gateway service
```

Comment:
I would move the Fabric Gateway section to just before the "Applications and Peers" section.
We need to establish the concepts of ledger and chaincode first in the next sections, before mentioning that Gateway service can manage the chaincode interactions on behalf of client applications.

Reply:
ok.